.github/workflows/live-test.yml

```diff
@@ -0,0 +1,39 @@
+name: Live Test
```

Comment:
Is this one needed to be separate? Can we not just use the main.yml with a manual or scheduled trigger?

Reply:
Main runs build, unit test and deploy.   We will have already run these for any PR.   We're only lacking the ability to run live tests against the PR.